pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1952,6 +1965,7 @@ static __always_inline int init_program_data(program_data_t *p, void *ctx)
 
     return 1;
 }
+// clang-format on
```

Comment:
?

Reply:
keep functions lesser than 100 cols without splitting in an unreadable way (use the stanza when you're sure the formatting is better than the auto formatted one).